send_comments.py

```
@@ -0,0 +1,82 @@
+import requests
+import os
+from dotenv import load_dotenv
+from app.utils.logger import logger
+
+load_dotenv()
+github_token = os.getenv("GITHUB_TOKEN_KEY")
+
+class GitHubCommenter:
+    def __init__(self, owner: str, repo: str, pr_number: int):
+        self.headers = {
+            "Authorization": f"Bearer {github_token}",
+            "Accept": "application/vnd.github+json"
+        }
+        self.owner = owner
+        self.repo = repo
+        self.pr_number = pr_number
+        self.issue_number = pr_number  # PR cũng là Issue
+
+    def send_general_comment(self, message: str):
+        url = f"https://api.github.com/repos/{self.owner}/{self.repo}/issues/{self.issue_number}/comments"
+        data = {
+            "body": message
+        }
+        response = requests.post(url, headers=self.headers, json=data)
```

Position 25: Medium severity: send_general_comment does not handle network errors or non-2xx responses. That can lead to unhandled exceptions or silent failures. Wrap requests.post in try/except, call response.raise_for_status(), and return/raise a meaningful exception on failure. Example:

try:
    response = requests.post(url, headers=headers, json=payload, timeout=10)
    response.raise_for_status()
except requests.RequestException as e:
    logger.exception("Failed to send general comment: %s", e)
    raise
return response.json()